source/gui/filebox.cpp

```diff
@@ -1278,17 +1290,22 @@ namespace nana
 			fd->Release();
 		}
 #else
-		BROWSEINFO brw = { 0 };
 		wchar_t	display_text[MAX_PATH];
-		brw.hwndOwner = reinterpret_cast<HWND>(API::root(impl_->owner));
-		brw.pszDisplayName = display_text;
-		brw.ulFlags = BIF_RETURNONLYFSDIRS | BIF_NEWDIALOGSTYLE;
-		brw.lpfn = browse_folder_callback;
-
+		auto title = to_wstring( impl_->title ) ;
 		std::wstring init_path = impl_->init_path.wstring();
-		brw.lParam = reinterpret_cast<LPARAM>(init_path.c_str());
-		auto pidl = ::SHBrowseForFolder(&brw);
 
+		// https://docs.microsoft.com/en-us/windows/desktop/api/shlobj_core/ns-shlobj_core-_browseinfoa
+        BROWSEINFO brw       = { 0 };
```

Comment:
Here, 8 spaces are used for indentation. Using 2 tabs might be good to keep consistency.